src/workers/newNotificationV2Mail.ts

```diff
@@ -717,8 +717,18 @@ const notificationToTemplateData: Record<NotificationType, TemplateDataFunc> = {
         id: notification.referenceId,
       });
 
+    const keyword = await userTopReader.keyword;
+    if (!keyword) {
+      throw new Error('Keyword not found');
+    }
+
     return {
-      topReader: userTopReader.image,
+      image: userTopReader.image,
+      issuedAt: new Date(userTopReader.issuedAt).toLocaleString('en-US', {
```

Comment:
`formatMailDate`?

Reply:
That would format `2024-10-31T00:00:00.000Z` as `Oct 31, 2024`. We want it to be `October 2024` to match format in front-end for the badges.

Reply:
![image](https://github.com/user-attachments/assets/aefdda79-516f-4834-8a4f-08120a4f5012)

Reply:
I thought they can format inside the templates

Reply:
Probably, it's just not done in this template. I could take a look

Reply:
It's possible. Will use `formatMailDate`